.snyk

```diff
@@ -0,0 +1,6 @@
+# Exclude specs from static analysis
+exclude:
+  global:
+    - examples
```

Comment:
These examples are 5 years old. Would it be better to remove them and document different examples in the readme?

Reply:
I am good with a removal of that, Readme already have all the details on how to setup sinatra and rails.